docs/_docs/01_getting-started/introduction.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: collection-browser-doc
+title: Introduction
+category: getting-started
+toc: true
+excerpt: >-
+  Terratest provides a variety of helper functions and patterns for common infrastructure testing tasks. Learn more about Terratest basic usage.
+tags: ["basic-usage"]
+order: 100
+nav_title: Documentation
+nav_title_link: /docs/
+---
+
+## Introduction
+
+Terratest is a Go library that makes it easier to write automated tests for your infrastructure code. It provides a
+variety of helper functions and patterns for common infrastructure testing tasks, including:
+
+- Testing Terraform code
+- Testing Packer templates
+- Testing Docker images
+- Executing commands on servers over SSH
+- Working with AWS APIs
+- Working with Azure APIs
+- Working with GCP APIs
+- Working with Kubernetes APIs
+- Testing Helm Charts
+- Making HTTP requests
+- Running shell commands
+- And much more
+
+For an introduction to Terratest, including unit tests, integration tests, end-to-end tests, dependency injection, test

```

Comment:
Since the next section is the same talk, and the "Open sourcing Terratest" blog post is a bit old now, I think this paragraph can be removed.